# HG changeset patch
# User Rik <rik@octave.org>
# Date 1515089251 28800
#      Thu Jan 04 10:07:31 2018 -0800
# Node ID 2bee24503a13d08d5edfebb730bf46f9bdfdae7d
# Parent  e61e26d1ede953fcfd02cf09ecbff66ffc333e84
maint: Correct some typos in code comments in color-picker.cc.

* color-picker.cc: Correct "bitton" -> "button", "selct" -> "select".

diff --git a/libgui/src/color-picker.cc b/libgui/src/color-picker.cc
--- a/libgui/src/color-picker.cc
+++ b/libgui/src/color-picker.cc
@@ -26,41 +26,41 @@ along with Octave; see the file COPYING.
 // Author: Torsten <ttl@justmail.de>
 
 #if defined (HAVE_CONFIG_H)
 #  include "config.h"
 #endif
 
 #include "color-picker.h"
 
-// constuctor with initial color as parameter
+// Constructor with initial color as parameter
 color_picker::color_picker (QColor old_color, QWidget *p)
   : QPushButton (p)
 {
   m_color = old_color;
   setFlat (true);
   setFocusPolicy (Qt::NoFocus);  // no focus, would changes the color
   update_button ();
   connect (this, SIGNAL (clicked (void)), SLOT (select_color (void)));
 }
 
-// slot for bitton clicked: selct a new color using QColorDialog
+// Slot for button clicked: select a new color using QColorDialog
 void
 color_picker::select_color (void)
 {
   QColor new_color = QColorDialog::getColor (m_color);
 
   if (new_color.isValid () && new_color != m_color)
     {
       m_color = new_color;
       update_button ();
     }
 }
 
-// draw the button with the actual color (using a stylesheet)
+// Draw the button with the actual color (using a stylesheet)
 void
 color_picker::update_button (void)
 {
   // Is this the right place to look for a "foreground" color that would
   // provide a reasonable border for the color swatches?
   QWidget *p = parentWidget ();
 
   QString bordercolor
